src/chart/custom/customSeriesRegister.ts

```diff
@@ -0,0 +1,11 @@
+import { CustomSeriesRenderItem } from './CustomSeries';
```

Comment:
Use `import type` explictly to ensure the `CustomSeries` is not in the depedencies of `core/echarts`
```ts
import type { CustomSeriesRenderItem } ...
```